include/cuco/detail/open_addressing/open_addressing_ref_impl.cuh

```diff
@@ -113,20 +120,76 @@ class open_addressing_ref_impl {
   /**
    * @brief Constructs open_addressing_ref_impl.
    *
+   * @param empty_key_sentinel The reserved key value for empty slots
    * @param empty_slot_sentinel Sentinel indicating an empty slot
+   * @param predicate Key equality binary callable
    * @param probing_scheme Probing scheme
    * @param storage_ref Non-owning ref of slot storage
    */
   __host__ __device__ explicit constexpr open_addressing_ref_impl(
+    cuco::empty_key<key_type> empty_key_sentinel,
```

Comment:
This is another cleanup I wanted to bring up: with ref having `has_payload`, the key sentinel is no longer necessary since the info is already contained in `empty_slot_sentinel_`

Reply:
To clarify, this can be deferred to another PR if you want.

Reply:
It's a small change so it fits in this PR. I'm working on it. BTW good idea! Much cleaner.

Reply:
Addressed in [dc71ba0](https://github.com/NVIDIA/cuCollections/pull/379/commits/dc71ba08aa81c942c0537795c228bb937c16b9ce)